stacker/actions/build.py

```diff
@@ -122,6 +138,9 @@ def _launch_stack(self, stack, **kwargs):
                                                      required_params,
                                                      provider_stack)
 
+        if not should_submit(stack):
```

Comment:
This seems a little late to test for this - any reason not to do it right at the beginning, since you already have the stack?